CMakeLists.txt

```diff
@@ -173,4 +173,46 @@ endif(HYDROCHRONO_ENABLE_TESTS)
 # Not a good idea to copy DLLs
```

Comment:
Comment to be removed

Reply:
That line does not come from this PR (cleaning it could be consider out of scope of this PR?)